src/io/flutter/server/vmService/VmServiceWrapper.java

```diff
@@ -204,6 +208,25 @@ public void received(Success response) {
     }
   }
 
+  public void attachIsolate(@NotNull IsolateRef isolateRef) {
+    //myDebugProcess.getSession().initBreakpoints(); // TODO(messick) Remove if not needed.
```

Comment:
you are calling initBreakpoints right before calling this method so I expect this isn't needed unless there is a separate code path.

Reply:
The are two paths to this code, and I think that's true for both.